NohanaImagePicker/PhotoKitAssetList.swift

```diff
@@ -19,7 +19,7 @@ import Photos
 open class PhotoKitAssetList: ItemList {
 
     fileprivate let mediaType: MediaType
-    open let assetList: PHAssetCollection
+    public let assetList: PHAssetCollection
```

Comment:
what was this warning? 
not sure Shopify will ever need to override this field but I think the picker lib was giving that option to the consumers.

Reply:
There's no reason to override a let since you can set it only once.